mobile/src/libs/components/button/button.tsx

```diff
@@ -15,7 +15,7 @@ type Properties = {
   label?: string;
   onPress: () => void;
   isDisabled?: boolean;
-  type?: 'solid' | 'outlined' | 'transparent';
+  type?: 'solid' | 'outlined' | 'transparent' | 'modal-cancel' | 'modal-delete';
```

Comment:
we need to simplify this and get rid of too specific variations.

1. we can't add margins for the common button component
2. in case we need to change some property for example color, let's add another property not variation

```suggestion
  type?: 'solid' | 'outlined' | 'transparent';
  color?: 'red' | 'blue';
```